util/file_util.cc

```diff
@@ -16,7 +16,8 @@ namespace rocksdb {
 
 // Utility function to copy a file up to a specified length
 Status CopyFile(Env* env, const std::string& source,
-                const std::string& destination, uint64_t size) {
+                const std::string& destination, uint64_t size,
+                const bool sync_after_copy) {
```

Comment:
By not having such a parameter, I mean always calling fsync. Just remove the if in line 66.